ansible_rulebook/app.py

```diff
@@ -141,6 +152,9 @@ async def run(parsed_args: argparse.ArgumentParser) -> None:
     await event_log.put(dict(type="Exit"))
     if error_found:
         raise Exception("One of the source plugins failed")
+    elif should_reload is True:
```

Comment:
Minor: This would be useful only if the variable can have other values different that an boolean that could be evaluated to `True` which is not the case as the variable is to store a boolean. The recommended practice would be:
```
elif should_reload:
```

Ref:
https://legacy.python.org/dev/peps/pep-0008/#programming-recommendations

Reply:
> Minor: This would be useful only if the variable can have other values different that an boolean that could be evaluated to `True` which is not the case as the variable is to store a boolean. 


Thank you for highlighting this, but I beg to differ: the `should_reload` returns the cancelled future in _some_ test cases, which is why it originally recursed infinitely (it went into this branch because cancelled, was valorized as a future) and I fixed it with 8d795da I believe

Reply:
Oh, sorry, you are right, I forgot the whole code from last week and I need a one more coffee :)

Reply:
no worries! 👍 🚀